smart-contracts/contracts/cl-vault/src/vault/swap.rs

```diff
@@ -218,6 +235,162 @@ fn execute_swap_operations(
     Ok(swap_msg)
 }
 
+pub fn execute_swap_non_vault_funds(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    force_swap_route: bool,
+    swap_routes: Vec<SwapAsset>,
```

Comment:
swap_routes is a confusing name here for the different assets that we want to swap